plugins/inputs/jboss/jboss.go

```diff
@@ -0,0 +1,1262 @@
+package jboss
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/plugins/common/tls"
+	"github.com/influxdata/telegraf/plugins/inputs"
+	dac "github.com/stefa975/go-http-digest-auth-client"
+)
+
+// Constants applied to jboss management query types
+const (
+	GetExecStat = iota
+	GetHosts
+	GetServers
+	GetDBStat
+	GetJVMStat
+	GetDeployments
+	GetDeploymentStat
+	GetWebStat
+	GetJMSQueueStat
+	GetJMSTopicStat
+	GetTransactionStat
```

Comment:
Those should be private variables starting with a lowercase letter. But I guess you don't need those if you switch to the variables scheme I suggested earlier.